### PR TITLE
Add UAVCAN on FMUv6X and Durandal

### PR DESCRIPTION
**Pull request by davids5**
> @david_s5 uavcan (v0) still disabled on px4_fmu-v6x and holybro_durandal?

@dagar - now enabled. Had to disable tests to fit, is there a better choice of things to disable?

### CODE REVIEW

**Review by LorenzMeier — DISMISSED** [review]

**Comment by LorenzMeier** [comment]
Thanks!

File: boards/holybro/durandal-v1/init/rc.board_defaults [hunk 1]
```diff
@@ -5,3 +5,5 @@
 
 # Enable IMU thermal control
 param set-default SENS_EN_THERMAL 1
+
+param set UAVCAN_ENABLE 2
```
Comment (dagar) [inline boards/holybro/durandal-v1/init/rc.board_defaults:9]: This should already be set by default on stm32h7. https://github.com/PX4/PX4-Autopilot/blob/master/platforms/nuttx/init/stm32h7/rc.board_arch_defaults

Comment (davids5) [reply boards/holybro/durandal-v1/init/rc.board_defaults]: @dagar - I missed that :) 

fixed in last push

**Comment by dagar** [comment]
> @dagar - now enabled. Had to disable tests to fit, is there a better choice of things to disable?

Probably most of the examples first. The test rack still mainly uses the default variants and runs the test. I think pretty soon we'll have to split out tests to completely separate builds.

**Comment by davids5** [comment]
> > @dagar - now enabled. Had to disable tests to fit, is there a better choice of things to disable?
> 
> Probably most of the examples first. The test rack still mainly uses the default variants and runs the test. I think pretty soon we'll have to split out tests to completely separate builds.

I think we are past that point.  All Examples except one are already gone and with it out, it still will not fit.

Please do the `test Pr config` and this can be re based on it.

**Comment by dagar** [comment]
Surprisingly big cpu boost as well (~8-10%).

### Master
Screen Shot 2021-04-02 at 6 13 01 PM

### PR

Screen Shot 2021-04-02 at 6 13 22 PM

**Review by LorenzMeier — APPROVED** [review]

**Pull request merged**